src/Backend/Modules/MediaLibrary/Resources/config/console.yml

```diff
@@ -1,10 +1,9 @@
 services:
-    media_library.console.delete_media_items:
-        class: Backend\Modules\MediaLibrary\Console\MediaItemDeleteAllCommand
+    media_library.console.cache_clear:
```

Comment:
I don't know which codes I should add to make this work.
After some searching I didn't find any sources about this...

Can you point me in the right direction please?